src/main/java/core/basesyntax/service/impl/ReportCreatorImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportCreator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportCreatorImpl implements ReportCreator {
+
+    public static final String TITLE = "fruit,quantity";
+    public static final String COMMA = ",";
+
+    @Override
+    public List<String> create(FruitStorage storage) {
```

Comment:
we can have a direct access to the storage instead of passing it as a parameter, if we use it's static map or pass some `StorageDao`(if we would have one) to the constructor of the `ReportCreator`